Create Film Festival website for module -1 capstone project

ram1117 opened the pull request:
1. The theme of the web page is a film festival
2. Create mobile versions of home page and about page
3.  Create desktop versions of home page and about page
4. Use javascript to create mobile navigation menu
5. Use javascript to add Jury members section in main page
6. Create readme file to document the project
7. Create project presentation video using loom. https://www.loom.com/share/62fba1a8a73c417a89da44272c292b1a

ram1117 committed:
setup project with readme,linters,main page and associated css file

1.Add microverse readme template to project
2.Add html-css-js linters to project
3.Add .gitignore file to project
4.Add main page and about page html files and css files
5.Add icons and background images to project

ram1117 committed:
Add programs section to main page

1.Add programs section and programs to main page
2.Update icons styling

ram1117 committed:
Add mark up and styling for  programs,jury section and partners in main page

1.Add mark up and styling for programs to programs section
2.Add partner logos resource folder
3.Modify icons used in the project

ram1117 committed:
Add mobile version of about page

1.Add markup and styling for about page mobile version
2.Add background images for about page mobile version

ram1117 committed:
Add desktop version for main page

1.Add desktop version of main page using media query in mainpagestyle.css
2.Alter icons and logos used in the main page for styling purposes

ram1117 committed:
Add desktop version to about page

1.Add desktop version to about page using media query in aboutpagestyle.css
2.Add dark theme footer logo and social media icons to resource folder

ram1117 committed:
modify icons and logos folder names

ram1117 committed:
Add javascript functionality for jury members section

ram1117 committed:
Add jury members section using javascript

1.Add cards to jury members section using javascript
2.Add image resources needed for jury member cards

ram1117 committed:
Fix linter errors in javascript and css style sheets

ram1117 committed:
Add popup menu window to main and about page mobile versions

ram1117 committed:
Tweak background images and body font size for responsive design

ram1117 committed:
Add breadcrumb navigation to about page

1.Add markup,style and icons for breadcrumb navigation in about page
2.Tweak font sizes and line heights in both pages
3.Add text contents to about page sections

ram1117 committed:
Fix stylelint errors in mainpagestyle and aboutpagestyle css files

ram1117 committed:
Make navigation menu fixed for mobile main page and about page

ram1117 committed:
Update raedme file in the project

ram1117 committed:
Update license terms in readme file of the project

frank1738 commented on stylesheets/mainpagestyle.css:
```diff
@@ -0,0 +1,550 @@
+}
+
+.header-welcome,
+.header-title {
+  color: #ec5242;
+  background-color: #f7f7f7;
+}
```
- [ ] You can make your header texts more appealing by giving it a background image and using ` -webkit-background-clip: text;` &`  -webkit-text-fill-color: transparent;` properties to match the original design

![image](https://user-images.githubusercontent.com/98453979/210373390-84d9208b-bcd2-4a0e-8ca4-9c629ec78255.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/210373599-5a58d684-100a-4278-ad99-4d57b2796343.png)

frank1738 commented on index.html:
```diff
@@ -0,0 +1,107 @@
+  <section class="section-intro">
+    <small class="header-welcome">"Hello! Vanakkam"</small>
+    <h1 class="header-title">TAMIL FILM ASSOC.<br>FILM FESTIVAL<br>2022</h1>
+    <p class="intro-description">Most Popular Film Festival in South India, gathering film talents from all
+      over the World. More than
+      20 films are to be screened throughout the week-long festival. And much more. </p>
```
- [ ] I would recommend that you reduce the width of this section and also adjust the `font-size` to match the original design
 
![image](https://user-images.githubusercontent.com/98453979/210374635-621bffcf-69e3-4318-8381-6bd13b5c77ce.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/210374842-944426ff-042e-4d27-b92f-720506596a13.png)

frank1738 commented on index.html:
```diff
@@ -0,0 +1,107 @@
+        <h4 class="program-title">Screenings</h4>
+        <p class="program-description">Over 20 films to be screened during the week</p>
+      </div>
+      <div class="program-tile">
+        <img src="bg-icons/workshop-icon.svg" alt="movie screening" class="program-icon">
+        <h4 class="program-title">Workshops</h4>
+        <p class="program-description">Workshops by eminent directors and technicians</p>
```
- [ ] Kindly implement the `hover` properties of your cards to match the original design

![image](https://user-images.githubusercontent.com/98453979/210375373-34de2513-7077-48b2-8e27-33c96da9d6bd.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/210375587-8ff80662-244b-42f0-8099-6bb14a8ef9e2.png)

frank1738 reviewed (changes requested):
Hi @ram1117 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media3.giphy.com/media/z5cNqbFbMTDA4/giphy.gif"/>


## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay
✅ Good PR Description👍 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ram1117 committed:
Make changes to implement code review 1  review comments

1.Add background to main page header text
2.Add hover effect to program tiles in program section
3.Add spacing between program tiles in mobile version
4.Modify width of main page introduction paragraph to match design

ram1117 committed:
fix stylelint error in mainpagestyle.css

pinkmoon25 commented on stylesheets/mainpagestyle.css:
```diff
@@ -116,25 +119,26 @@ body {
-  font-size: 0.8em;
-  font-weight: 500;
+  font-size: 0.9em;
+  font-weight: 700;
```
- [ ] [OPTIONAL] Good job implemeting the changes requested by the previous reviewer, however the cards in mobile view still look a bit busy, I think we can improve it by setting justify-content on .program-title to space between and removing the bold property for font weight and by reducing the width property to say 40-45% , how it looks now :point_down: 
![image](https://user-images.githubusercontent.com/97521557/210400524-1245baba-b979-48e7-b10d-095b0f829d07.png)

how it could be :point_down:
 
![image](https://user-images.githubusercontent.com/97521557/210400822-39f44e19-9f54-455c-acab-f42b0295fd50.png)

ram1117 replied:
@PinkMoon25 hi Tushar, thanks for this feedback. I ll implement the suggestion.

pinkmoon25 reviewed (approved):
Hi @ram1117 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<img src="https://media1.giphy.com/media/3o7abKhOpu0NwenH3O/giphy.gif"/>

## Highlights
Correct use of Github workflow :heavy_check_mark: 
No linter error :heavy_check_mark: 
README is okay :heavy_check_mark: 
 Good PR Description :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ram1117 committed:
fix program tile width based on code review 2 comment
